### ISSUE TITLE
Copy not derived for opaque array

### ISSUE THREAD

**Issue by polachok** [jsgf/rust-xdr#20]
In libvirt's [remote protocol](https://libvirt.org/git/?p=libvirt.git;a=blob_plain;f=src/remote/remote_protocol.x;hb=HEAD) there're such structures:

(VIR_UUID_BUFLEN == 16)

```
typedef opaque remote_uuid[VIR_UUID_BUFLEN];

struct remote_domain_lookup_by_uuid_args {
    remote_uuid uuid;
};
```
xdrgen doesn't derive(Copy) on remote_uuid 

```
# [ derive ( Debug , Eq , PartialEq , Clone , Copy ) ] pub struct remote_domain_lookup_by_uuid_args { pub uuid : remote_uuid , }

# [ derive ( Debug , Eq , PartialEq , Clone ) ] pub struct remote_uuid ( pub [ u8 ; VIR_UUID_BUFLEN as usize ] ) ;
```
leading to compilation errors:

```
error[E0204]: the trait `Copy` may not be implemented for this type
   --> /home/plhk/src/libvirt-rpc/target/debug/build/libvirt-rpc-bf9fddb9810869f9/out/remote_protocol_xdr.rs:691:47
    |
691 | # [ derive ( Debug , Eq , PartialEq , Clone , Copy ) ] pub struct remote_domain_lookup_by_uuid_args { pub uuid : remote_uuid , }
    |                                               ^^^^                                                    ---------------------- this field does not implement `Copy`
```

This can be worked around by adding `impl Copy for remote_uuid {}`, no big problem, just wanted to let you know.

**Comment by jsgf** [comment]
Hm, OK.

**Comment by jsgf** [comment]
I can't repro this with:
```
const VIR_UUID_BUFLEN = 16;

typedef opaque remote_uuid[VIR_UUID_BUFLEN];
struct remote_domain_lookup_by_uuid_args {
    remote_uuid uuid;
};
```

It generates
```
#[derive( Copy , Clone , Debug , Eq , PartialEq )] pub struct remote_uuid ( pub [ u8 ; VIR_UUID_BUFLEN as usize ] ) ;
```
as expected.

But it looks like in the full remote_protocol.x, `VIR_UUID_BUFLEN` is not defined in the file, so xdrgen can't see the definition and assumes that it might be >32, which would prevent it from trying to derive `Copy` for the type. The fix is to make sure the definition is visible to xdrgen.

**Comment by polachok** [comment]
```
const VIR_UUID_BUFLEN = 16;

typedef opaque remote_uuid[VIR_UUID_BUFLEN];
struct remote_domain_lookup_by_uuid_args {
    remote_uuid uuid;
};
```

```
xdrgen test.x

// GENERATED CODE
//
// Generated from test.x.
//
// DO NOT EDIT


# [ derive ( Debug , Eq , PartialEq , Clone , Copy ) ] pub struct remote_domain_lookup_by_uuid_args { pub uuid : remote_uuid , }

# [ derive ( Debug , Eq , PartialEq , Clone ) ] pub struct remote_uuid ( pub [ u8 ; 16i64 as usize ] ) ;
```

**Comment by jsgf** [comment]
What version are you running? Is it from git or crates.io?

**Comment by polachok** [comment]
v0.4.0 from crates.io

**Comment by jsgf** [comment]
I published 0.4.1 with recent fixes/changes.
